### PR TITLE
Make it clear which tool is selected in control panel

### PR DESCRIPTION
**Pull request by Tornquist**
# :green_heart: Summary

The selected tool will now be highlighted in the control panel.  Each tool will be highlighted in a different color.  I pulled the colors from bootstrap.

The default line width was increased to 5.0.  This will make it easier to see what is drawn, and will make the brush control much clearer in the control panel.

The new highlight icons are used in the launch screen as well.

### CODE REVIEW

**Comment by e7711bbear** [comment]
Conflicts, please resolve :)

**Comment by Tornquist** [comment]
Resolved.

File: ATSketchKit/ATSketchView.swift [hunk 1]
```diff
@@ -22,19 +22,19 @@
 
 import UIKit
 
+public enum Tools {
```
Comment (e7711bbear) [inline ATSketchKit/ATSketchView.swift]: Please leave the tools inside the view. It makes more sense as they are only used inside the ATSketchView.
And yes, I know this means changing everything else :stuck_out_tongue_closed_eyes:

**Comment by Tornquist** [comment]
Tools is no longer externally accessible.  I added some new button colors and a splash screen as part of the transition as well.

File: ATSketchKitDemo/ViewController.swift [hunk 2]
```diff
@@ -146,5 +170,11 @@ class ViewController: UIViewController, ATSketchViewDelegate, ATColorPickerDeleg
 		return nil
 	}
 
+    // MARK: - UI Updates
+    func highlightControlButton(selectedButton: UIButton) {
```
Comment (e7711bbear) [inline ATSketchKitDemo/ViewController.swift]: Please change the name to reflect the toggling behavior.

**Comment by Tornquist** [comment]
Sounds good. You can expect this change sometime on the 14th.

**Comment by Tornquist** [comment]
This should be ready to go now.

**Comment by e7711bbear** [comment]
Looks good. :+1:

**Pull request merged**